src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -312,6 +318,13 @@ public void execute() throws GitException, InterruptedException {
                 warnIfWindowsTemporaryDirNameHasSpaces();
 
                 launchCommandWithCredentials(args, workspace, cred, url, timeout);
+
+                if (withLFS) {
+                    ArgumentListBuilder lfsArgs = new ArgumentListBuilder();
+                    lfsArgs.add("lfs");
```

Comment:
Since it is an explicit setting that one would turn on, it makes most sense to me to just fail if it isn't present, rather than just log it and continue the build (which will mostly likely fail in some not-well-defined way, when it, for example, tries to extract a zip file that is not a zip file since lfs).

If you would rather thIs do a check, is there a pattern in the git client plugin I can reference for checking for git feature support?